src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -305,7 +342,7 @@ internal static FormulaValue ExpandDateTimeExcelFormatSpecifiers(IRContext irCon
                     }
                     catch (FormatException)
                     {
-                        return CommonErrors.GenericInvalidArgument(irContext, StringResources.Get(TexlStrings.ErrTextInvalidFormat, culture.Name));
+                        return null;
```

Comment:
>return null; [](http://example.com/codeflow?start=24&length=12)

Can we follow TryGet pattern , and avoid returning null. 
That makes it very explicit.

Reply:
Updated this.